src/components/SearchSelectInput/EmptyOptions/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import styles from './styles.css';
+
+interface DefaultEmptyComponentProps {
+    optionsPending?: boolean;
+    isFiltered?: boolean;
+}
+
+function EmptyOptions(props: DefaultEmptyComponentProps) {
+    const {
+        isFiltered = false,
```

Comment:
@tnagorra might argue that this is not right way of naming a boolean variable. But, I'm fine with it :P